contracts/SpokePool.sol

```diff
@@ -754,10 +754,12 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
             fillAmountPreFees = amountRemainingInRelay;
 
             // The user will fulfill the remainder of the relay, so we need to compute exactly how many tokens post-fees
-            // that they need to send to the recipient.
+            // that they need to send to the recipient. Note that if the relayer is filled using contract funds then
+            // this is a slow relay. In this case the depositor should get refunded the relayerFeePct. To do this we
+            // simply exclude it from the fees taken from the fillAmountPreFees, thereby refunding the user this amount.
             amountToSend = _computeAmountPostFees(
                 fillAmountPreFees,
-                relayData.realizedLpFeePct + updatableRelayerFeePct
+                relayData.realizedLpFeePct + (useContractFunds ? 0 : updatableRelayerFeePct)
```

Comment:
Should we emit the `relayerFeePct` as "0" in the `FilledRelay` event? Or should we keep it as is which implicitly says that  we are emitting the original `relayerFeePct` set by the depositor, not the actual one used in production.

FWIW I lean towards setting `relayerFeePct` to 0 and then renaming the param in the `FilledRelay` event from `relayerFeePct` to `appliedRelayerFeePct`

Reply:
@mrice32 we do the same thing in the speed up case where we emit the updated relay fee pct

Reply:
Awesome. SGTM